packages/cashier/src/Containers/deposit.jsx

```diff
@@ -42,8 +42,6 @@ const Deposit = ({
     setSideNotes,
     tab_index,
 }) => {
-    const is_crypto = !!currency && isCryptocurrency(currency);
```

Comment:
@likhith-deriv Lets don't remove `is_crypto` and replace the condition for showing `CryptoDeposit` with this.